util/mqtt.go

```diff
@@ -33,17 +33,21 @@ const (
 	MqttCmdPrefix = "cmnd/"
 )
 
-func init() {
-	conf := config.GetConfig()
-	hostPort := net.JoinHostPort(conf.MqttDomain, strconv.FormatInt(int64(conf.MqttPort), 10))
-	uri := fmt.Sprintf("tcp://%s", hostPort)
-
-	opts := mqtt.NewClientOptions().AddBroker(uri)
-	opts.SetClientID(conf.ValidatorAddress)
-	opts.SetUsername(conf.MqttUser)
-	opts.SetPassword(conf.MqttPassword)
-	MQTTClient = mqtt.NewClient(opts)
+func LazyLoadMQTTClient() {
```

Comment:
Can be local (lowercase "l"): `lazyLoadMQTTClient`

Reply:
yes